src/sagemaker/jumpstart/constants.py

```diff
@@ -132,3 +145,29 @@
 ENV_VARIABLE_JUMPSTART_SPECS_LOCAL_ROOT_DIR_OVERRIDE = "AWS_JUMPSTART_SPECS_LOCAL_ROOT_DIR_OVERRIDE"
 
 JUMPSTART_RESOURCE_BASE_NAME = "sagemaker-jumpstart"
+
+
+CONTENT_TYPE_TO_SERIALIZER_TYPE_MAP: Dict[MIMEType, SerializerType] = {
+    MIMEType.X_IMAGE: SerializerType.RAW_BYTES,
+    MIMEType.LIST_TEXT: SerializerType.JSON,
+    MIMEType.X_TEXT: SerializerType.TEXT,
+    MIMEType.JSON: SerializerType.JSON,
+    MIMEType.CSV: SerializerType.CSV,
+    MIMEType.WAV: SerializerType.RAW_BYTES,
+}
+
+
+ACCEPT_TYPE_TO_DESERIALIZER_TYPE_MAP: Dict[MIMEType, DeserializerType] = {
+    MIMEType.JSON: DeserializerType.JSON,
+}
+
+SERIALIZER_TYPE_TO_CLASS_MAP: Dict[SerializerType, Callable[[], BaseSerializer]] = {
```

Comment:
technically, it's still a callable though, since it's invoked with the `__call__` method. But i'll change the typing since it's unnecessarily complicated.